vnext/ReactWindowsCore/ReactWindowsCore.vcxproj

```diff
@@ -4,4 +4,5 @@
     <ProjectGuid>{11C084A3-A57C-4296-A679-CAC17B603144}</ProjectGuid>
   </PropertyGroup>
   <Import Project="ReactWindowsCore.vcxitems" Label="Shared" />
+  <Target Name="Deploy" />
```

Comment:
><Target Name="Deploy" /> [](start = 2, length = 24)

Why do we need empty targets?

Reply:
@vmoroz Without empty targets, the solution will fail to build (asking a project to build an empty target is an error)